src/mpc/structs.jl

```diff
@@ -308,6 +342,154 @@ struct MPCGenerator <: AbstractGenerator
     end
 end
 
+#HYDROGEN TECHS
+"""
+    MPCElectrolyzer
+
+```julia
+Base.@kwdef struct MPCElectrolyzer < AbstractElectrolyzer
+    size_kw::Float64
+    efficiency_kwh_per_kg::Float64 =  54.3
+    om_cost_per_kwh::Float64 = 0.006
+end
+```
+"""
+Base.@kwdef struct MPCElectrolyzer <: AbstractElectrolyzer
+    size_kw::Float64
+    efficiency_kwh_per_kg::Float64 =  54.3
+    om_cost_per_kwh::Float64 = 0.006
+end
+
+"""
+    MPCHydrogenStorageLP
+
+```julia
+Base.@kwdef struct MPCHydrogenStorageLP < AbstractHydrogenStorage
+    size_kg::Float64
+    soc_min_fraction::Float64 = 0.01
+    soc_init_fraction::Float64 = 0.5
+    daily_leakage_fraction::Float64 = 0.0006667
+    max_kg::Float64 = size_kg
+    minimum_avg_soc_fraction::Float64 = 0.0
```

Comment:
how would a non zero value for this be enforced in MPC? the function where this is used (add_elec_storage_dispatch_constraints) has 8760 hard coded, assuming there are a full year of time steps. something we need to think about more for MPC